ui/widget.js

```diff
@@ -514,7 +514,7 @@ $.Widget.prototype = {
 				// - disabled class as method for disabling individual parts
 				if ( !suppressDisabledCheck &&
 						( instance.options.disabled === true ||
-							$( this ).hasClass( "ui-state-disabled" ) ) ) {
+						$( this ).hasClass( "ui-state-disabled" ) ) ) {
```

Comment:
This is an interesting one. I guess we don't have any rules about multi-line groups, so this is correct according to the style guide. I've started splitting closing parens on to their own lines in my projects because of this.